things/api/things/http/transport.go

```diff
@@ -169,6 +169,13 @@ func MakeHandler(tracer opentracing.Tracer, svc things.Service) http.Handler {
 		opts...,
 	))
 
+	r.Delete("/disconnect", kithttp.NewServer(
+		kitot.TraceServer(tracer, "disconnect_list")(disconnectListEndpoint(svc)),
```

Comment:
This naming is good but I'd also rename `create_connections` into `connect_list` to be consistent. Maybe even replacing old endpoints and use the naming `connect` and `diconnect`  for this new endpoints that use lists.